doc/content/lorawan/antenna-connectors/troubleshooting.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Troubleshooting Antenna Connectors"
```

Comment:
what is this?

Reply:
It is an auto-generated file. I will delete it.